metaflow/plugins/argo/argo_workflows.py

```diff
@@ -234,14 +316,21 @@ def _process_parameters(self):
                     "Scheduling such parameters via Argo CronWorkflows is not "
                     "currently supported." % param.name
                 )
-            value = deploy_time_eval(param.kwargs.get("default"))
-            # If the value is not required and the value is None, we set the value to
-            # the JSON equivalent of None to please argo-workflows.
-            if not is_required or value is not None:
-                value = json.dumps(value)
-            parameters.append(
-                dict(name=param.name, value=value, description=param.kwargs.get("help"))
-            )
+            if include_values:
```

Comment:
nit - `self.parameters` is already available to `SensorTemplate` - we can ignore the values set for parameters and save a duplicate evaluation of parameter names.